src/Http/Request.php

```diff
@@ -89,11 +89,12 @@ public function getParameters()
 
     /**
      * @param string $name
+     * @param null $default
      * @return mixed
      */
-    public function getParameter(string $name)
+    public function getParameter(string $name, $default = null)
     {
-        return $this->parameters[$name];
+        return array_key_exists($name, $this->getParameters()) ? $this->parameters['name'] : $default;
```

Comment:
Why `getParameters()` in condition, but then direct `$this->parameters` in true clause ?

Reply:
I end up using the alternative: return $this->parameters['name'] ?? $default;